src/modules/jackrack/lv2_plugin.c

```diff
@@ -272,8 +274,19 @@ static int lv2_plugin_instantiate(const LilvPlugin *plugin,
                                   gint copies,
                                   LilvInstance **instances)
 {
-    gint i;
+  char *lv2context_can_ui = mlt_environment ("lv2context_can_ui");
+  if (lv2context_can_ui != NULL)
+    {
+      /* Video editors and other hosts that support custom GUI should use mlt_environment_set ("lv2context_can_ui", "1")
+	 to inform mlt lv2 plugin manager and set UI features and extensions if not set.
+       */
+      if (lv2context_can_ui[0] == '1')
+	{
+	  //WIP: if support UI
+	}
+    }
```

Comment:
For hosts (ie. Video Editors) that want to support LV2 Custom GUIs they should set the mlt environment variable "lv2context_can_ui" to "1" and the the lv2 plugin instance will be instantiated with LV2 UI features and extensions and then they can get "global_lv2_world" handle of LilvWorld* and and retrieve the UI URI and handles and create ui support with suil library (http://drobilla.net/software/suil.html).

This is the plan for supporting Custom GUIs.